src/gui/fixconflictingfilesdialog.cpp

```diff
@@ -209,9 +211,9 @@ QString FixConflictingFilesDialog::descriptionText() const {
 void FixConflictingFilesDialog::insertFileItems(const int nbItems) {
     uint64_t max = std::min(_fileListWidget->count() + nbItems, (int)_conflictList.size());
     for (uint64_t i = _fileListWidget->count(); i < max; i++) {
-        FileItemWidget *w = new FileItemWidget(_conflictList[i].destinationPath(), _conflictList[i].nodeType(), this);
+        auto w = new FileItemWidget(_conflictList[i].destinationPath(), _conflictList[i].nodeType(), this);
```

Comment:
```suggestion
        auto w = new FileItemWidget(_conflictList[i].destinationPath(), _conflictList[i].nodeType());
```

Reply:
Same, but there may be a memory leak because of the instantation of the `QListWidgetItem`. Should be fixed now